charms/kfp-persistence/src/templates/auth_manifests.yaml.j2

```diff
@@ -54,5 +54,7 @@ subjects:
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: {{ app_name }}-sa
+  labels:
+    app: {{ app_name }}
+  name: {{ sa_name }}
```

Comment:
just a question, why is this needed?

Reply:
What part, the whole ServiceAccount?

Reply:
the removal of `-sa` from the service account name

Reply:
Oh, well that is because I'm keeping the ServiceAccount name the same as in upstream, see [here](https://github.com/canonical/kfp-operators/pull/349/files#diff-30a280bd44c1418d68f3c82b68265bc3bffec6bf25d23c23e342ab71e09ecf38R35) and [here](https://github.com/kubeflow/manifests/blob/master/apps/pipeline/upstream/base/pipeline/ml-pipeline-persistenceagent-sa.yaml#L4)